src/components/AsyncRouter/AsyncRouter.js

```diff
@@ -9,38 +9,50 @@ import Browse from '../Browse/Browse';
 import Preface from '../Preface/Preface';
 import Navbar from '../Navbar/Navbar';
 import DummyNavbar from '../PreloadNavbar/PreloadNavbar';
+import I18n from '../I18n';
 
 const DICTIONARY_VERSION = 'woodhouse-0.0.2';
 
+const base = '/:locale(en|fr)?';
+
 const Router = ({ basename, dictionary }) => (
   <BrowserRouter basename={basename}>
     <React.Fragment>
       <Switch>
-        <Route path="/(l|b)/:word?" component={Navbar} />
-        <Route path="*" component={Navbar} />
+        <Route path={`${base}/(l|b)/:word?`} component={Navbar} />
+        <Route path={`${base}*`} component={Navbar} />
       </Switch>
       <div className="container text-center">
-        <header>
-          <h1 className="h3 pt-4 mb-1 font-weight-normal">
-            English-Greek Dictionary
-          </h1>
-          <h4 className="h5 mb-2 font-weight-normal">
-            <em>
-              A Vocabulary of the Attic Language
-            </em>
-          </h4>
-          <h5 className="h5 mb-3 font-weight-normal">
-            S.C. Woodhouse
-          </h5>
-        </header>
+        <Route
+          path={base}
+          render={() => (
+            <header>
+              <h1 className="h3 pt-4 mb-1 font-weight-normal">
+                <I18n t="header.title" />
+              </h1>
+              <h4 className="h5 mb-2 font-weight-normal">
+                <em>
+                  <I18n t="header.subtitle" />
+                </em>
+              </h4>
+              <h5 className="h5 mb-3 font-weight-normal">
+                <I18n t="header.author" />
+              </h5>
+            </header>
+          )}
+        />
         <main>
           <div className="mb-4">
             <Switch>
-              <Route exact path="/" render={props => <Lookup {...props} dictionary={dictionary} />} />
-              <Route exact path="/l/:word?" render={props => <Lookup {...props} dictionary={dictionary} />} />
-              <Route exact path="/b/:word?" render={props => <Browse {...props} dictionary={dictionary} />} />
-              <Route exact path="/p/preface" component={Preface} />
-              <Route exact path="/:word?" render={props => <Lookup {...props} dictionary={dictionary} />} />
+              <Route
```

Comment:
It looks like the other `<Route>` below are on one line but this one is spread across several lines. Is there a reason for doing it this way? If not, let's move it to one line to match the others.

Reply:
This particular line was setting off the linter for being too long. I can leave it and set eslint to ignore, or keep it extended.

Reply:
Since this line is shorter than the other ones, it seems like a bug in the linter. For now, let's move it to one line and change `path={base}` to ``path={`${base}`}``. This should make the linter be quiet.